frontend/src/main/pages/UCSBDates/UCSBDatesEditPage.js

```diff
@@ -12,7 +12,10 @@ export default function UCSBDatesEditPage() {
     useBackend(
       // Stryker disable next-line all : don't test internal caching of React Query
       [`/api/ucsbdates?id=${id}`],
-      { method: "GET", url: `/api/ucsbdates?id=${id}` }
+      {  // Stryker disable next-line all : GET is the default, so changing this to "" doesn't introduce a bug
+        method: "GET",
+        url: `/api/ucsbdates?id=${id}`
```

Comment:
Can we use axios `params` here? Something like
```
{
  method: "GET",
  url: "/api/ucsbdates",
  params: {
    id,
  },
}
```

Manually composing a URL is unsafe and can lead to issues with unescaped input, so it's best we make it common practice to use axios to form the query string instead.